packages/components/src/lozenge/tests/chromatic/Lozenge.chroma.jsx

```diff
@@ -0,0 +1,218 @@
+import { PlaceholderMajorIcon, IconList } from "@components/icons";
```

Comment:
This file should not be added. Its been renamed to .stories.tsx and this file added is the result of a merge conflict gone wrong